schema/v5.0/CVE_JSON_5.0.schema

```diff
@@ -975,6 +975,28 @@
                     "value": {
                         "type": "string",
                         "minLength": 1
+                    },
+                    "user": {
+                        "type": "string",
+                        "description": "UUID of the user if present in the CVE User Registry",
+                        "maxLength": 64
```

Comment:
This should have a UUID pattern. Something like

`"pattern" : "^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-4[0-9A-Fa-f]{3}-[89ABab][0-9A-Fa-f]{3}-[0-9A-Fa-f]{12}$"`

Reply:
There are multiple fields that use UUID in the schema. I'll create a definition for UUID fields (with a pattern) that abstracts this data type. I'll do this in a separate PR. You can update this PR once the UUID one is merged.

Reply:
done, referenced to user UUID definition.